packages/coinstac-ui/app/render/components/consortium/consortium-results.js

```diff
@@ -19,14 +19,15 @@ class ConsortiumResults extends Component {
     setExpandedResult(initialResultId);
   }
 
-  componentDidMount() {
-    this.goToResults();
-  }
-
   goToResults() {
-    if (document.querySelector('#results')) {
-      document.querySelector('#results').scrollIntoView();
-    }
+    const { expandedResult } = this.props;
+
+    // Hacky - used to delay querying ids until results list items are rendered
+    setTimeout(() => {
+      if (document.querySelector(`#C${expandedResult}`)) {
+        document.querySelector(`#C${expandedResult}`).scrollIntoView();
+      }
+    }, 500);
   }
```

Comment:
I used setTimeout here to wait to query the result id until after the results list items were rendered. I'm sure there is a better way and would welcome some advice.

Also, the appended 'C' in the ID is to avoid an error being thrown when using _querySelector_ on an ID that can begin with a digit. I learned today that is a no, no.